static/src/js/routes/Admin/Admin.jsx

```diff
@@ -63,12 +64,20 @@ export class Admin extends Component {
                   >
                     View Retrieval Metrics
                   </PortalLinkContainer>
+                  |
+                  {' '}
+                  <PortalLinkContainer
+                    to="/admin/preferencesMetrics"
+                  >
+                    View Preferences Metrics
+                  </PortalLinkContainer>
                 </Route>
                 <Route exact path={`${path}/retrievals`} component={AdminRetrievalsContainer} />
                 <Route exact path={`${path}/retrievals/:id`} component={AdminRetrievalContainer} />
                 <Route exact path={`${path}/projects`} component={AdminProjectsContainer} />
                 <Route exact path={`${path}/projects/:id`} component={AdminProjectContainer} />
                 <Route exact path={`${path}/retrievalsMetrics`} component={AdminRetrievalsMetricsContainer} />
+                <Route exact path={`${path}/preferencesMetrics`} component={AdminPreferencesMetricsContainer} />
```

Comment:
It looks like you copied what `retrievalsMetrics` is doing, but both are wrong. URLs are supposed to by hyphenated, see the URLs defined in App.jsx, https://github.com/nasa/earthdata-search/blob/main/static/src/js/App.jsx#L137

Reply:
Should this be converted on a different ticket then? Feels like changing retrievalMetrics is out of scope for this and keeping things consistent is good. It would be a very quick fix and would take little to no resources.